framework_tvb/tvb/adapters/analyzers/fourier_adapter.py

```diff
@@ -37,22 +37,30 @@
 """
 import math
 import uuid
-
 import numpy
 import psutil
-import tvb.analyzers.fft as fft
 from tvb.adapters.datatypes.db.spectral import FourierSpectrumIndex
 from tvb.adapters.datatypes.db.time_series import TimeSeriesIndex
 from tvb.adapters.datatypes.h5.spectral_h5 import FourierSpectrumH5
+from tvb.analyzers.fft import compute_fast_fourier_transform
 from tvb.core.adapters.abcadapter import ABCAdapterForm, ABCAdapter
 from tvb.core.entities.filters.chain import FilterChain
 from tvb.core.neocom import h5
 from tvb.core.neotraits.forms import ScalarField, TraitDataTypeSelectField, SelectField
 from tvb.core.neotraits.view_model import ViewModel, DataTypeGidAttr
 from tvb.datatypes.time_series import TimeSeries
+from tvb.basic.neotraits.api import HasTraits, Attr, Float, narray_describe
+
+
+SUPPORTED_WINDOWING_FUNCTIONS = {
```

Comment:
We could leave this in the library, since FourierSpectrum expects a string for `windowing_function` attribute. Pls review this once more to asses what is the best way to use it,